src/Telegram.Bot/Requests/Abstractions/Documentation.cs

```diff
@@ -78,4 +78,4 @@ internal abstract class Documentation
     /// Protects the contents of sent messages from forwarding and saving
     /// </summary>
     object ProtectContent;
-}
\ No newline at end of file
+}
```

Comment:
`insert_final_newline = true` in .editorconfig?

Reply:
Already present, just forgot to enable it in Rider to enforce it.